_build/header.html.erb

```diff
@@ -47,8 +46,10 @@ section#format-index .row {
 }
 
   </style>
+<!--
   <script src="<%= @base_url %>js/highlight.pack.js"></script>
   <script>hljs.initHighlightingOnLoad();</script>
+-->
```

Comment:
Given that we're switching from one to another, it makes sense to remove highlight.js altogether.
```suggestion
```

Reply:
Of course, thanks! It's just a forgotten comment.